src/app/shared/services/white-label.service.ts

```diff
@@ -10,6 +10,21 @@ export class WhiteLabelService {
 
   isWhiteLabelSubdomain(): boolean {
     const hostname = this.document.location.hostname;
-    return hostname === WHITE_LABEL_HOSTNAME;
+    return WHITE_LABEL_CONFIGS.some(config => config.hostname === hostname);
```

Comment:
This looks the same as getCurrentWhiteLabelConfig

Reply:
This will return true or false. The `getCurrentWhiteLabelConfig` returns the entire config object, bro.

Reply:
Okay bro.